packages/react-core/src/components/Tooltip/Tooltip.tsx

```diff
@@ -97,6 +98,19 @@ export interface TooltipProps extends Omit<React.HTMLProps<HTMLDivElement>, 'con
 // id for associating trigger with the content aria-describedby or aria-labelledby
 let pfTooltipIdCounter = 1;
 
+function isButtonOrIcon(children: React.ReactNode) {
+  if (React.isValidElement(children)) {
+    if (['button', Button as any, 'svg', 'g'].includes(children.type)) {
```

Comment:
Checkbox, Radio we also use tooltips to 'explain' why elements are disabled. 

So when we disable a checkbox, we always wrap it with a tooltip and say 'Please blabla to enable this feature'